### DISCUSSION TITLE
Next Workshop

### DISCUSSION THREAD

**Discussion by zklaus** [ESMValGroup/Community#87] [Polls]
Dear @ESMValGroup/esmvaltool-developmentteam, everyone interested,

the next ESMValTool workshop will be held hybrid, with the on-site part hosted at SMHI in Norrköping, Sweden.
It will be held in May or June. To determine the best week, please fill in the poll at https://framadate.org/9ycbFxIPzLbEMmEt.

The workshop will take three days, from Tuesday to Thursday, allowing for travel and side meetings on Monday and Friday.

The workshop is open to the entire ESMValTool community and will contain introductory level as well as in-depth discussions.

It also is a great opportunity to meet fellow ESMValTool developers and get to know the community!

If you plan to participate remotely, please let us know your timezone, so we can plan an inclusive schedule.

We are looking forward to meeting you!

**Comment by valeriupredoi** [comment]
The official mascot of the workshop, guys :sweden: 

![SAAB-Sk-35C-Draken](https://user-images.githubusercontent.com/28983971/232101060-f1e03878-0690-49f8-8f46-7625ecf1eae4.jpg)

**Comment by bouweandela** [comment]
@zklaus Is there a date for the workshop already? The people at the monthly meeting https://github.com/ESMValGroup/Community/discussions/97 were asking about it.

**Reply by zklaus** [reply]
Yes, the workshop will take place 2023-06-27 to 2023-06-29. I'll make a new post to announce it.
